docker_helper.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -o errexit
+set -o nounset
+
+if [ $# -ne 4 ]
+then
+    echo "Usage: $0 [TUMOR_BAM] [NORMAL_BAM] [REFERENCE] [OUTPUT_DIR]"
+    exit 255
+fi
+
+TUMOR_BAM=$1
+NORMAL_BAM=$2
+REFERENCE=$3
+OUTPUT_DIR=$4
+
+$STRELKA_INSTALL_DIR/bin/configureStrelkaWorkflow.pl --tumor=$TUMOR_BAM --normal=$NORMAL_BAM --ref=$REFERENCE --output-dir=$OUTPUT_DIR --config=$STRELKA_INSTALL_DIR/etc/strelka_config_bwa_default.ini
```

Comment:
Yeah, the sed-like substitution above is an example of the parameter and how the value is updated.  I think it's fine to require a config as input.  The config files are shipped with the strelka code as a convenience.  It doesn't seem unreasonable to ask a user of this image to store that config and provide it as an input.  In the same way, should we add an optional input for [extraStrelkaArguments|https://sites.google.com/site/strelkasomaticvariantcaller/home/configuration-and-analysis/configuration-extrastrelkaoptions]?